src/base_object-inl.h

```diff
@@ -32,10 +32,21 @@
 
 namespace node {
 
+namespace {
+// This just has to be different from the Chromium ones:
+// https://source.chromium.org/chromium/chromium/src/+/main:gin/public/gin_embedders.h;l=18-23;drc=5a758a97032f0b656c3c36a3497560762495501a
+// Otherwise, when Node is loaded in an isolate which uses cppgc, cppgc will
+// misinterpret the data stored in the embedder fields and try to garbage
+// collect them.
+static uint16_t kNodeEmbedderId = 0x90de;
```

Comment:
```suggestion
uint16_t kNodeEmbedderId = 0x90de;
```
nit: This is already in an anonymous namespace, so there's no need for the static keyword.